README.md

```diff
@@ -1194,6 +1194,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Hitesh Pal](https://github.com/Hs809)
   - [Hithesh Jay](https://github.com/hith3sh)
   - [Hizqeel Nizamani](https://github.com/ezekielnizamani)
+  - [Himanshu Sharma](https://github.com/mighty-uno)
```

Comment:
Hey, in order to keep the list alphabetically correct, please insert your name between those two entries:
   [Himanshu Parihar]
   [Himanshu Singh Chauhan]
(this is currently line 1192)